scripts/system_tests_utils.ts

```diff
@@ -165,7 +165,7 @@ export const closeOpenConnection = async (web3Context: Web3Context) => {
 		(web3Context.provider as unknown as Web3BaseProvider).disconnect();
 
 		await new Promise(resolve => {
-			setTimeout(resolve, 1000);
+			setTimeout(resolve, 1);
```

Comment:
what was the idea behind waiting for 1s here? the more tests use this function to close connection, the longer the whole test suite will execute.

Reply:
It was waiting for ws to reset and disconnect for random ws connection not closing in tests.